CONTRIBUTING.org

```diff
@@ -81,6 +81,7 @@ Submit your contribution against the =develop= branch. You should not use
 your =master= branch to modify Spacemacs, this branch is considered to be
 read-only.
 
+You may want to [[https://github.com/syl20bnr/spacemacs/wiki/Beginner%2527s-Guide-to-Contributing-a-Pull-Request-to-Spacemacs][read our beginner guide for Pull Requests]]. 
```

Comment:
Without a new line before `PR=Pull Request` they are in the same paragraph/line. Is this in purpose?